### PR TITLE
Story/FBC Empty State/1130

### PR DESCRIPTION
**Pull request by conbrad**
https://wps-pr-1142.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator

- Empty state has "select a" user prompt label
- Investigated empty state label misalignment but it's not clear that MaterialUI supports styling this easily + this current table is temporary
- Remove the "X" clear button from select drop downs as they do not work

### CODE REVIEW

File: web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx [hunk 1]
```diff
@@ -143,16 +157,23 @@ const FBCInputGrid = (props: FBCInputGridProps) => {
...
 
+    const finalVal: GridMenuOption = buildGridMenuOption(value, optionBoxType)
+    const label = isEqual(finalVal, theEmptyOption)
+      ? optionBoxType.type === 'station'
+        ? 'Select a station'
+        : 'Select a fuel type'
+      : undefined
     return (
       <Autocomplete
         id={`combo-box-fuel-types-${Math.random()}`}
```
Comment (Sybrand) [inline web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx:168]: Is there a better way than just grabbing a random number?

**Review by tessmgood — APPROVED** [review]

**Review by Sybrand — CHANGES_REQUESTED** [review]
Issue:
If I remove the grass cure on O1A/O1B, and press calculate - nothing appears to happen for the user. (In the background there's a 500 error from the API).

We used to have validation to ensure that you couldn't send an empty grass cure down - what's happened to that?

Nice to have:
Can you have a look at some of the code smells and address them where easy / practical? https://sonarcloud.io/project/issues?id=bcgov_wps&pullRequest=1142&resolved=false&types=CODE_SMELL

File: web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx [hunk 2]
```diff
@@ -64,7 +60,7 @@ const GrassCurePercentageEdit = (props: NumberEditProps) => {
       id="grass-cure-percentage-number"
       type="number"
       value={props.value}
-      required={true}
+      required={false}
```
Comment (Sybrand) [inline web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx:63]: Isn't this supposed to be true for O1A and O1B?

**Comment by conbrad** [comment]
Going to defer merging this in favor of the better results from: https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/1141

**Comment by andrea-williams** [comment]
> Going to defer merging this in favor of the better results from: https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/1141

In that case, I'll hold off on even reviewing this one

**Pull request closed without merging**